### DISCUSSION TITLE
libopenmpt clang

### DISCUSSION THREAD

**Discussion by revelator** [msys2/MINGW-packages#9669] [Q&A]
running into a somewhat weird problem with this one.
the library itself builds but the console player fails to link with an undefined reference to wWinMain@16 when built with clang.
i tried all the fixes in the book and it is not working :S.

**Comment by raedrizqie** [comment]
try adding `-municode` to CPPFLAGS maybe?

**Reply by revelator** [reply]
allready added still fails :S

**Comment by raedrizqie** [comment]
Do you happens to mistakenly put `-mwindows` rather than `-mconsole`?

**Reply by revelator** [reply]
no but the source does put -mwindows on the compiler commands which was also a hint to me that something was up.
looks like we need a patch.

**Reply by raedrizqie** [reply]
ok then
